### ISSUE TITLE
Unable to get transactions from Tangerine (Canada)

### ISSUE THREAD

**Issue by spiffylogic** [euforic/banking.js#66]
Here is what I get:

```
{ SIGNONMSGSRSV1:
   { SONRS:
      { STATUS: 
          { CODE: '0',
            SEVERITY: 'INFO',
            MESSAGE: 'Authentication Successful.' },
        DTSERVER: '20180624232758.373[-4:EDT]',
        LANGUAGE: 'ENG',
        FI: { ORG: 'TangerineBank', FID: '10951' } } },
  BANKMSGSRSV1:
   { STMTTRNRS:
      { TRNUID: 'CrtFKOellYoIErC0YT2Lmdg8zGNmljoY',
        STATUS: { CODE: '2003', SEVERITY: 'ERROR' },
        CLTCOOKIE: 'TN9fO' } } }
```

So it looks like it authenticated successfully, however something went wrong with BANKMSGSRSV1 (error code 2003). 

I wonder if this would have anything to do with the security questions tangerine asks when connecting from a new device.

Or maybe I need to try different headers? Not sure how to do that or what values to try.

Any tips much appreciated.

**Comment by spiffylogic** [comment]
I also tried this with my credit card account and got successful authentication (as above), but also this error below. At list it has a message!
```{ CCSTMTTRNRS:
   { TRNUID: 'tT4adpDq1ScrJdp4iDiO1Gp7fdyx4tn3',
     STATUS:
      { CODE: '2000',
        SEVERITY: 'ERROR',
        MESSAGE: 'The server does not support the CREDITCARDMSGSRQV1 request, or a request was made against an invalid organization.  Please verify the Organization name and FID and try again.\r\n[ref: C0A8D7874140F30C0000016439C31201061D]' },
     CLTCOOKIE: 'wKQt0' } }
```

**Comment by euforic** [comment]
Closing since this is an issue with banks support. Feel free to reopen if you find a bug.

**Comment by Joshfindit** [comment]
Also going for Tangerine here.

The error `2003` is "Account Not Found" according to https://microsoftmoneyoffline.wordpress.com/money-ofx-error-codes-messages/

Once I found that, I pulled the account list and found that **all** accounts are `"ACCTTYPE": "SAVINGS"`, so when I was specifying `accType: 'CHECKING'` is was (apparently) not found.

Similarly, the Tangerine credit card is `"ACCTTYPE": "CREDITLINE"`.

Using both these tweaks, I was able to get transactions from each account.
